vnext/src/Libraries/Network/RCTNetworkingWinShared.js

```diff
@@ -19,7 +19,14 @@ import type {RequestBody} from './convertRequestBody';
 
 class RCTNetworking extends NativeEventEmitter {
   constructor() {
-    super(RCTNetworkingNative);
+    const disableCallsIntoModule =
+      typeof global.__disableRCTNetworkingExtraneousModuleCalls === 'function'
+        ? global.__disableRCTNetworkingExtraneousModuleCalls()
```

Comment:
>__disableRCTNetworkingExtraneousModuleCalls [](start = 17, length = 43)

We probably don't control the name, but having a function named `disableXYZ` reads as an operation, not a property inspection. `isXYZDisabled` would make more sense for property inspection...

Reply:
Yeah, this whole thing is an iOS only hack. Not sure of the motivations of it, but keeping our patches in sync.